New package: BEACN.BEACNApp version 1.1.137.0

AffinityNa opened the pull request:
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.10 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.10.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

AffinityNa committed:
New package: BEACN.BEACNApp version 1.1.137.0

AffinityNa commented:
I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer

AffinityNa commented:
> @AffinityNa please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree

AffinityNa committed:
Update BEACN.BEACNApp.installer.yaml

local testing showed that while the app dictated /exenoui to work, it did in fact still launch with a UI.. so I've changed it to the MSIExec option and that worked locally.

stephengillie commented:
Automatic Validation ended with:
>  The FileSystemWatcher has detected an error System.IO.ErrorEventArgs

(Automated response - build 1048.)

AffinityNa committed:
Update BEACN.BEACNApp.installer.yaml

passive was giving a warning about unplugging the hardware for drivers to install properly. quiet suppresses this.

AffinityNa commented:
> Automatic Validation ended with:
> 
> > The FileSystemWatcher has detected an error System.IO.ErrorEventArgs
> 
> (Automated response - build 1048.)

This should be resolved now.. I think. This is my first time contributing, so I hope I'm correct here. I ran it locally and did get a popup mid install, now when I run it I no longer get that.

ItzLevvie reviewed (changes requested):
![image](https://github.com/user-attachments/assets/e8724fa2-ad3a-42b7-84ba-e84db1d80e7e)

ItzLevvie commented:
[Policy] Reset Labels
[Policy] Reset Feedback

ItzLevvie commented:
[Policy] Needs-Author-Feedback

ItzLevvie commented:
Close with reason: Cleaning up old pull requests. If your pull request was closed by mistake please ping me so I can reopen your pull request;
